deploy/operator.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
The operator should be a DaemonSet, right?

Reply:
Very good point, just changed it to DaemonSet.